samples/bluetooth/scanner/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: TBD
+  name: TBD
```

Comment:
So when do you intend to write something here, if not now ?

Reply:
I have given the name, removed the description field as it's already available in the README file.